### ISSUE TITLE
Sim is zoomed in upon loading

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/curve-fitting#166]
For https://github.com/phetsims/QA/issues/431.

I don't know if this was just a fluke, but I opened the sim in portrait mode on iOS 12.4.1 (Hopper), and the sim was "zoomed in." I put "zoomed in" in quotation marks because I couldn't actually zoom back out. I don't know if this is a browser issue or a scenery issue or an issue that is specific to this simulation. As you can probably tell, I haven't found steps to reproduce. Just documenting in case it comes up again, which seems unlikely.

![issue](https://user-images.githubusercontent.com/31285590/65641698-b3790d00-dfaa-11e9-8f94-3d1bbff315b4.PNG)

**Comment by SaurabhTotey** [comment]
Hmmm... That is odd. I haven't seen this happen, but I'll be sure to document here if I observe it happening.

**Comment by jbphet** [comment]
@KatieWoe and @lmulhall-phet - this issue was unassigned, and we're getting close to publication of this sim, so I'm assigning it to you to see if you can find a way to reliably reproduce it using the latest RC, which is https://phet-dev.colorado.edu/html/curve-fitting/1.0.0-rc.5/phet/curve-fitting_en_phet.html.

**Comment by jbphet** [comment]
There is a desire to get this sim published so that it can go out in the next release of the iOS app, so I'm going to assume that this is not a publication-blocking issue and @SaurabhTotey and I will move forward with initial publication.  @KatieWoe and @lmulhall-phet - we'd still like to know if this can be duplicated and if so, and if it can be fixed, we will plan to address it in a maintenance release.  If not, please close and we'll just keep an eye out for it.

**Comment by KatieWoe** [comment]
Yeah.  I've tried and haven't had any luck so far.  Sorry about that.  We'll keep an eye out.
